modules/upgrade/pages/k-rolling-upgrade.adoc

```diff
@@ -56,6 +52,12 @@ If your current version of Redpanda is more than one feature release behind the
 +
 To find information about what has changed between Redpanda versions, check the https://github.com/redpanda-data/redpanda/releases[release notes^].
 
+== Check license status
+
+If you use xref:get-started:licensing/overview.adoc#self-managed[enterprise features] of Redpanda, make sure that you have a valid Enterprise Edition license key. Redpanda fails to upgrade to a new feature release if you have enterprise features enabled without a valid license key.
```

Comment:
Would an "as of version 24.3 Redpanda will fail to upgrade if" disclaimer be useful here?

It may also be good to specify that the failure is due to redpanda crashing or refusing to startup which could have an performance impact on a running cluster. Otherwise I could see this being interpreted as "Oh, I just won't get to use 24.3's features if my license isn't valid"

Reply:
```suggestion
If you use xref:get-started:licensing/overview.adoc#self-managed[enterprise features] of Redpanda, make sure that you have a valid Enterprise Edition license key. From version 24.3, Redpanda fails to upgrade to a new feature release if you have enterprise features enabled without a valid license key.
```